.stylelintrc

```diff
@@ -0,0 +1,8 @@
+{
+  "rules": {
+    "selector-class-pattern": "mapbox[a-z-]+",
+    "color-hex-case": "lower",
```

Comment:
I'm not sure we need the color rules. we rarely change `dist/mapbox-gl.css` and we don't deal with much color in there.